utils/src/bulk-insert-patients.ts

```diff
@@ -60,7 +63,14 @@ const mapCSVPatientToMetriportPatient = (csvPatient: {
   zipcode: string;
   phone: string;
   gender: string;
-}): PatientCreate => {
+}): PatientCreate | undefined => {
+  const validGender = csvPatient.gender === "Male" || csvPatient.gender === "Female";
+
+  if (!csvPatient.gender || !validGender) {
```

Comment:
hypernit: could this just be `if(!validGender)`?

Reply:
True will fix